controllers/booking.js

```diff
@@ -60,18 +60,20 @@ async function getOneById(req, res) {
 }
 
 // @desc    Create a booking
-// @route   POST /booking/create/:restaurantId
+// @route   POST /booking/create
```

Comment:
this code worked for me but didn't return an errorneous message when I entered an invalid restaurantID (ie, restaurant id "65a278d1f5ae0f9372509410" doesn't exist in my restaurant dummy data)
<img width="871" alt="image" src="https://github.com/jx0906/proj3-backend/assets/142247158/4430ca54-166d-47c0-b3c5-edafcd7778d8">
<img width="865" alt="image" src="https://github.com/jx0906/proj3-backend/assets/142247158/1ee6b82e-3d12-4c93-9d82-396a6e97f661">

Reply:
@jx0906 
Apology! Modified.
Whatever error status returned by `modelRestaurant.getRestaurantById` will be caught. 
In case `modelRestaurant.getRestaurantById` returns `null` instead of error, new error will be generated in `createBooking` logic itself.

Reply:
no worries, just tested. it works well now! pls proceed to merge! think @cedricyong00 can review later on if he has time!